wiki/People/The_Team/Global_Moderation_Team/th.md

```diff
@@ -3,36 +3,43 @@ tags:
+  - chat mod
 ---
 
 # ทีมผู้ดูแล
 
-**Global Moderation Team (ทีมผู้ดูแล)** (เดิมชื่อ Global Mod Team, Administrators หรือ GreenBAT) โดยทั่วไปเรียกว่า **GMT** เป็นสมาชิกของชุมชนที่มุ่งเน้นในการดูแลผู้เล่นเป็นหลัก ประเด็นที่เกี่ยวข้องกับทีม และชุมชนโดยรวม
+*ดูเพิ่มเติม: [Mod (แก้คำกำกวม)](/wiki/Disambiguation/Mod).*
 
-**Global Moderator ทุกคน ได้สาบานที่จะทำตาม[จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
+**Global Moderation Team (ทีมผู้ดูแล)** (เดิมชื่อ Global Mod Team, Administrators หรือ GreenBAT) โดยทั่วไปเรียกว่า **GMT** เป็นสมาชิกของชุมชนที่มุ่งเน้นในการดูแลผู้เล่นเป็นหลัก รวมถึงประเด็นที่เกี่ยวข้องกับทีม และชุมชนโดยรวม พวกเขาสังเกตได้จากตำแหน่งชื่อผู้ใช้ และสีของ username ซึ่งเป็นสีแดงในเกมและสีเขียวซิตรัสบนฟอรัม
 
-หากคุณพบเห็นสมาชิกคนใดของ Global Moderation Team มีพฤติกรรมที่ไม่เหมาะสม โปรดติดต่อ [abuse@ppy.sh](mailto:abuse@ppy.sh) โดยทันที พร้อมข้อมูลมากที่สุดเท่าที่เป็นไปได้ สำหรับรายละเอียดเพิ่มเติม ดูที่ [รายงานการละเมิด](/wiki/Reporting_Bad_Behaviour/Abuse)
+**Global Moderator ทุกคน ได้สาบานที่จะทำตาม[จรรยาบรรณผู้ช่วยเหลือ](/wiki/Contributor_Code_of_Conduct) นอกเหนือจาก[กฎชุมชน](/wiki/Rules)**
 
 ## หน้าที่และความรับผิดชอบ
 
-ทีมผู้ดูแลมีหน้าที่รับผิดชอบด้านสวัสดิการของการแชท ฟอรัม และดูแลการมีปฏิสัมพันธ์ภายในชุมชน พวกเขาควรได้รับการติดต่อด้วยคำถามใด ๆ ที่เกี่ยวข้องกับแง่มุมการกลั่นกรองของเกมซึ่งรวมถึง (แต่ไม่จำกัดเฉพาะ) ปัญหาเกี่ยวกับการแชท (เช่น ความเงียบ หรือการล่วงละเมิด) และฟอรัม (เช่น การสแปม หรือความคิดเห็นที่ไม่เหมาะสม)
+***หมายเหตุ: หากคุณพบเห็นสมาชิกผู้ดูแลคนใดมีพฤติกรรมที่ไม่เหมาะสม หรือละเมิดจรรยาบรรณผู้ช่วยเหลือ โปรดติดต่อ [ทีมช่วยเหลือผู้เล่น](/wiki/People/The_Team/Account_support_team#support@ppy.sh)***
+*ดูเพิ่มเติม: [รายงานการละเมิด](/wiki/Reporting_Bad_Behaviour/)*
+
+ทีมผู้ดูแลมีหน้าที่รับผิดชอบด้านสวัสดิการของการแชท ฟอรัม และดูแลการมีปฏิสัมพันธ์ภายในชุมชน พวกเขาควรได้รับการติดต่อด้วยคำถามใด ๆ ที่เกี่ยวข้องกับแง่มุมการกลั่นกรองของเกมซึ่งรวมถึง (แต่ไม่จำกัดเฉพาะ):
+
+- ปัญหาเกี่ยวกับการแชท (เช่น การสแปม หรือการคุกคาม)
+- พฤติกรรมที่ไม่เหมาะสมบนฟอรัม (เช่น โพสที่ไม่เหมาะสม)
+- การพูดคุยในบีทแมพ และส่วนคอมเมนต์
+- การกรั่นกรองโปรไฟล์ผู้ใช้ (เนื้อหา NSFW (18+) หรือเนื้อหาที่ล่วงละเมิดในรูปผู้ใช้ รูปปกโปรไฟล์ รายละเอียด ฯลฯ)
 
 ## การเป็นสมาชิกทีมผู้ดูแล
 
-การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีการสนับสนุนอย่างมาก สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชน และมีพฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษนั้นเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
+การมาเป็นสมาชิกของทีมผู้ดูแลนั้น จำเป็นต้องมีผลงานการสนับสนุนในเกมอย่างเห็นได้ชัด สมาชิกที่มีศักยภาพมักจะถูกสังเกตจากที่ว่าพวกเขาเต็มใจที่จะช่วยเหลือผู้อื่น มีกิจกรรมภายในชุมชน และมีพฤติกรรมที่ดี ความสามารถในการพูดภาษาอื่นที่นอกเหนือจากภาษาอังกฤษนั้นเป็นที่ต้องการอย่างยิ่ง ผู้สมัครจะถูกพูดคุยภายในทีมเพื่อพิจารณาศักยภาพของพวกเขาในฐานะสมาชิกทีม
```

Comment:
activity -> กิจกรรม

there's probably a better word for this that hasn't came to my mind yet

Reply:
การเข้าร่วมในกิจกรรมของชุมชน? 
มีส่วนร่วมในกิจกรรมของชุมชน?

Reply:
@C0NS03L เหมือนใน version อังกฤษ คำว่า activity จะหมายถึงว่าแบบ active ในเชิงแบบได้มีประวัติแชท / รีพอร์ต / etc. ไม่ได้แบบว่ากิจกรรม literally อ่ะ อธิบายยาก

Reply:
I'm out of idea then haha. I guess กิจกรรม is good enough.